Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs

```diff
@@ -626,4 +626,40 @@ public bool TryFTLProximity(EntityUid shuttleUid, ShuttleComponent component, En
 
         return true;
     }
+
+    /// <summary>
+    /// Flattens / deletes everything under the grid upon FTL.
+    /// </summary>
+    private void Smimsh(EntityUid uid, FixturesComponent? manager = null, MapGridComponent? grid = null, TransformComponent? xform = null)
```

Comment:
make it gib bodies

Reply:
+1 for "Smish" - also did you spell that right?

Reply:
smimsh is intended.